src/main/webapp/app/exercises/shared/exercise/exercise.service.ts

```diff
@@ -122,7 +122,6 @@ export class ExerciseService {
     hasExampleSolutionPublicationDateError(exercise: Exercise) {
         if (exercise.exampleSolutionPublicationDate) {
             return (
-                !dayjs(exercise.exampleSolutionPublicationDate).isValid() ||
```

Comment:
this line is generally a nice validation, but should be done not only for this date and should not be done for exams.
This PR reverts this wrong validation. I will do a followup PR to readd date validation in a proper way.